graphql-kotlin-types/src/main/kotlin/com/expediagroup/graphql/types/GraphQLKotlinError.kt

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.types
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonValue
+import graphql.ErrorClassification
+import graphql.GraphQLError
+import graphql.language.SourceLocation
+
+/**
+ * GraphQL error representation that is spec complaint with serialization and deserialization,
+ * but still implements the graphql-java interface.
+ *
+ * @see [GraphQL Specification](http://spec.graphql.org/June2018/#sec-Errors) for additional details
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+open class GraphQLKotlinError(
```

Comment:
wondering whether we need both basic implementation of `GraphQLError` for deserializing the response in the client (i.e. current error from the client package) and one that extends it and also contains the underlying exception information (i.e. current error from spring server package).